### PR TITLE
Too many 'verifying' log messages, already above

### PR DESCRIPTION
**Pull request by mkrizek**

### CODE REVIEW

File: migrate.py [hunk 1]
```diff
@@ -1544,9 +1544,6 @@ def assert_migrating_git_tracked_resources(
     :param migrated_to_collection: Iterable of paths relative \
                                    to the ansible/ansible root.
     """
-    logger.info(
```
Comment (webknjaz) [inline migrate.py]: Maybe turn it into debug?

Comment (mkrizek) [reply migrate.py]: What is the gain of having this general message when we have already more specific one one level above? Also we can't turn debug off.

Comment (webknjaz) [reply migrate.py]: It's called from two places. The other one doesn't have an extra log on top. Also, logging into the file has warning level set.

Comment (mkrizek) [reply migrate.py]: Hm, is there a case where the function would pass on first call and not on the second one? Can we remove the second call to the function?

Comment (webknjaz) [reply migrate.py]: So when I first wrote it, I've been meaning to validate what's actually being migrated. The run checks the schema but over the course of migration extra files are added/auto-discovered.

Think of these as entry and exit checks, like in contract programming: you want to validate the input but you also want to validate that the end-result is also correct.

Comment (mkrizek) [reply migrate.py]: Forgot about auto-discovery. Updated.

File: migrate.py [hunk 2]
```diff
@@ -1561,6 +1558,7 @@ def assert_migrating_git_tracked_resources(
 def mark_moved_resources(checkout_dir, namespace, collection, migrated_to_collection):
     """Mark migrated paths in botmeta."""
     migrated_to_collection = {str(k): str(v) for k, v in migrated_to_collection.items()}
+    logger.info('Verifying that only git-tracked files are being migrated...')
```
Comment (webknjaz) [inline migrate.py]:
```suggestion
    logger.info(
        'Verifying that only git-tracked files are being migrated to "%s.%s"...',
        namespace, collection,
    )
```

**Pull request merged**